src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/CurrentAllocationController.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.keyworker.controllers
+
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.keyworker.config.MANAGE_ALLOCATIONS
+import uk.gov.justice.digital.hmpps.keyworker.dto.CurrentPersonStaffAllocation
+import uk.gov.justice.digital.hmpps.keyworker.services.GetCurrentAllocations
+
+@RestController
+class CurrentAllocationController(
+  private val allocations: GetCurrentAllocations,
+) {
+  @Tag(name = MANAGE_ALLOCATIONS)
+  @PreAuthorize("hasRole('${Roles.ALLOCATIONS_RO}')")
+  @GetMapping("/prisoners/{personIdentifier}/allocations/current")
```

Comment:
Ah totally missed this in the ACs. This will need adding to the `SentryConfig.isHighUsage` function so we don't overuse our quota